llvm/test/Transforms/Util/loop-iter-count-asserts.ll

```diff
@@ -0,0 +1,298 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+;
+; This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+; See https://llvm.org/LICENSE.txt for license information.
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+;
+; (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+; unit test for the loop iteration metadata conversion to assumptions
+;
+; NOTE: Example file for converting loop iter count to assumptions in the Loop
+;
+; RUN: opt -S -passes='loop-iter-count-metadata' < %s | FileCheck %s
```

Comment:
add -verify-each?

Reply:
yes that is a helpful option. i'll add it